BeerOverflowWindowsApp/BarFileReader.cs

```diff
@@ -12,13 +14,18 @@ static class BarFileReader
 
         public static BarDataModel GetAllBarData()
         {
-            BarDataModel result = null;
+             BarDataModel result = new BarDataModel();
+
             if (File.Exists(_filePath))
             {
-                var barsData = File.ReadAllText(_filePath);
-                result = JsonConvert.DeserializeObject<BarDataModel>(barsData);
+                List<string> barsDataJson = new List<string>(File.ReadAllLines(_filePath));
+                foreach(string bar in barsDataJson)
+                    result.Add(JsonConvert.DeserializeObject<BarData>(bar)); 
+            }
```

Comment:
How does this work? If i understand correctly you want to read file line by line and deserialize them into BarData objects.  
Why do you need to do that instead of just deserializing it all at once? Also are you sure that one line will be exactly one BarData?

Reply:
Well our practice lecturer said, than we should write and read separate objects instead of just one big object.
Yes i'm sure because that's the way i write them in file.